source/extensions/filters/udp/udp_proxy/udp_proxy_filter.h

```diff
@@ -342,10 +350,7 @@ class UdpProxyFilter : public Network::UdpListenerReadFilter,
 
   const UdpProxyFilterConfigSharedPtr config_;
   const Upstream::ClusterUpdateCallbacksHandlePtr cluster_update_callbacks_;
-  // Right now we support a single cluster to route to. It is highly likely in the future that
-  // we will support additional routing options either using filter chain matching, weighting,
-  // etc.
-  absl::optional<ClusterInfoPtr> cluster_info_;
+  absl::flat_hash_map<std::string, ClusterInfoPtr> cluster_infos_;
```

Comment:
nit: Please comment on what the key in this map is. (Cluster name?)

Reply:
Ok, done.